### PR TITLE
Add version: KDE.Kdenlive version 23.08.4

### LINKED ISSUE #141108: [Package Issue]: KDE.Kdenlive 23.08.4 -> 23.08.5
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

I have Kdenlive installed 23.08.5, winget shows it's 23.08.4 and offers me an update.

### Steps to reproduce

I have Kdenlive installed 23.08.4
Do winget upgrade KDE.Kdenlive
Installation finishes
winget still shows that I have 23.08.4 installed and there is a new version ready.
I check version in Kdenlive in the About window, it shows 23.08.5 version.

I uninstalled Kdenlive and reinstalled with winget, still having the same issue.

### Actual behavior

winget keeps offering me the 23.08.5 update

### Expected behavior

winget doesn't offer me an update, because I already have 23.08.5 installed.

### Environment

```raw
Windows Package Manager v1.6.3482
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22621.3155
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.21.3482.0

Winget Directories
-----------------------------------------------------------------------------------------------------------------------
Logs                               %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\Diag…
User Settings                      %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\sett…
Portable Links Directory (User)    %LOCALAPPDATA%\Microsoft\WinGet\Links
Portable Links Directory (Machine) C:\Program Files\WinGet\Links
Portable Package Root (User)       %LOCALAPPDATA%\Microsoft\WinGet\Packages
Portable Package Root              C:\Program Files\WinGet\Packages
Portable Package Root (x86)        C:\Program Files (x86)\WinGet\Packages
Installer Downloads                %USERPROFILE%\Downloads

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale

Admin Setting                             State
--------------------------------------------------
LocalManifestFiles                        Disabled
BypassCertificatePinningForMicrosoftStore Disabled
InstallerHashOverride                     Disabled
LocalArchiveMalwareScanOverride           Disabled

Other software:
WingetUI
```

### Screenshots and Logs

image
image

### PR DESCRIPTION
**Pull request by Exorcism0666**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

* Resolves #141108

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/142105)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Review by stephengillie — COMMENTED** [review]
Hi @Exorcism0666,

The InstallerUrl has the subsequent version number in it.

**Comment by Exorcism0666** [comment]
@stephengillie This is normal, if you try to install this version, normally it will always be indicated as 23.08.4

**Comment by stephengillie** [comment]
Verify:
image

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @Exorcism0666,

This PackageUrl led to a very different InstallerUrl. Should it be used instead?

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Specified hash doesn't match.
 Actual hash:   83C3B30DDF2298916A2FEF39CEE2CB2579764E6B99FE908EFE8CDB3CD12B8846
 Expected hash: 8712b9483669b5c01d5642681c6d9f1d191ff8b46848a29344dc873e0c92e138
 
 3/6/2024 6:30:25 PM
 
 ****************************************************************************************
 Installer Validation
 ****************************************************************************************
 

(Automated response - build 866.)

File: manifests/k/KDE/Kdenlive/23.08.4/KDE.Kdenlive.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: KDE.Kdenlive
+PackageVersion: 23.08.4
+InstallerType: nullsoft
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.kde.org/stable/kdenlive/23.08/windows/kdenlive-23.08.5.exe
+  InstallerSha256: 8712B9483669B5C01D5642681C6D9F1D191FF8B46848A29344DC873E0C92E138
```
Comment (stephengillie) [inline manifests/k/KDE/Kdenlive/23.08.4/KDE.Kdenlive.installer.yaml:11]:
```suggestion

```

(Automated response - build 866.)

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
> Possible duplicate package entry.
> 
> Similar installer SHA256 hash found in manifest:
> 
> * `InstallerSha256: 83c3b30ddf2298916a2fef39cee2cb2579764e6b99fe908efe8cdb3cd12b8846`
>   
>   * manifests\k\KDE\Kdenlive\23.08.05

Since the installer with `23.08.5` in the name installs the version in this PR, the manifest for `23.08.5` should be removed.

**Comment by Exorcism0666** [comment]
@trenly

**Pull request merged**